contrib/curl-cmake/CMake/CurlSymbolHiding.cmake

```diff
@@ -0,0 +1,61 @@
+include(CheckCSourceCompiles)
```

Comment:
Need to mention source and license. It can be made in README in this directory.
https://github.com/curl/curl/blob/3b8bbbbd1609c638a3d3d0acb148a33dedb67be3/CMake/CurlSymbolHiding.cmake